instrumentation/opentelemetry-instrumentation-cherrypy/tests/test_cherrypy.py

```diff
@@ -100,17 +111,25 @@ def healthzz(self):
             @cherrypy.expose
             def error(self):
                 raise cherrypy.HTTPError(500, 'error')
+            
+            @cherrypy.expose
+            def check_header(self):
+                cherrypy.response.headers["custom-test-header-1"]="test-header-value-1"
+                cherrypy.response.headers["custom-test-header-2"]="test-header-value-2"
+                content = {"message": "hello world"}
+                return content
 
-        return cherrypy.tree.mount(CherryPyApp())
+        cherrypy.tree.mount(CherryPyApp())
```

Comment:
why this change?
Even if we are not using the output we can still return it and ignore it.

Reply:
currently we only need to pass the cherrypyApp object to mount function the return is not used anywhere

Reply:
yes true. Return object is an CherryPy app so you can return it as it may get used later.

Reply:
ok i am doing that